src/config.ts

```diff
@@ -37,14 +37,65 @@ const clientConfig: ClientProps = {
     true
   ),
   enableLogging: envValueToBoolean(process.env.REACT_APP_OIDC_LOGGING, false),
-  tokenExchangePath: process.env.REACT_APP_OIDC_TOKEN_EXCHANGE_PATH
+  tokenExchangePath: process.env.REACT_APP_OIDC_TOKEN_EXCHANGE_PATH,
+  path: '/helsinkimvp',
+  routeMatch: '/helsinkimvp/:subpage?',
+  hasApiTokenSupport: true,
+  label: 'Helsinki-profiili MVP'
 };
 
 const uiConfig: { profileUIUrl: string } = {
   profileUIUrl: String(process.env.REACT_APP_PROFILE_UI_URL)
 };
 
+const plainSuomiFiConfig: ClientConfig = {
+  realm: String(process.env.REACT_APP_PLAIN_KEYCLOAK_REALM),
+  url: String(process.env.REACT_APP_PLAIN_KEYCLOAK_URL),
+  authority: process.env.REACT_APP_PLAIN_KEYCLOAK_REALM
+    ? `${process.env.REACT_APP_PLAIN_KEYCLOAK_URL}/realms/${process.env.REACT_APP_PLAIN_KEYCLOAK_REALM}`
+    : String(process.env.REACT_APP_PLAIN_KEYCLOAK_URL),
+  clientId: String(process.env.REACT_APP_PLAIN_KEYCLOAK_CLIENT_ID),
+  callbackPath: String(process.env.REACT_APP_PLAIN_KEYCLOAK_CALLBACK_PATH),
+  logoutPath: process.env.REACT_APP_PLAIN_KEYCLOAK_LOGOUT_PATH || '/',
+  silentAuthPath: process.env.REACT_APP_PLAIN_KEYCLOAK_SILENT_AUTH_PATH,
+  responseType: process.env.REACT_APP_PLAIN_KEYCLOAK_RESPONSE_TYPE,
+  scope: process.env.REACT_APP_PLAIN_KEYCLOAK_SCOPE,
+  autoSignIn: envValueToBoolean(
+    process.env.REACT_APP_PLAIN_KEYCLOAK_AUTO_SIGN_IN,
+    true
+  ),
+  automaticSilentRenew: envValueToBoolean(
+    process.env.REACT_APP_PLAIN_KEYCLOAK_AUTO_SILENT_RENEW,
+    true
+  ),
+  enableLogging: envValueToBoolean(
+    process.env.REACT_APP_PLAIN_KEYCLOAK_LOGGING,
+    false
+  ),
+  tokenExchangePath: '*not-supported*',
```

Comment:
Shouldn't it be the one who provides the configuration, who decides if `tokenExchangePath` is supported or not? Also, this magic string is still `truthy`, so in the `getTokenUri` function, where this value is used in an `if` clause, the condition there evaluates to `true`. I'm not sure if that's intended or a bug, as I'm confused with that function too, but there's another comment for that.

Reply:
tokenExchangePath would not been used in routes that do not support ApiAccessTokens. This in now obsolete anyway.